### PR TITLE
OdroidXU3: linux: revert kernel to 4.14 to fix gbm libmali

### PR DESCRIPTION
**Pull request by by-gnome**

### CODE REVIEW

**Comment by Ntemis** [comment]
Did you build the kernel successful?

**Comment by by-gnome** [comment]
Yes, but mali-midgard failed.

**Comment by ToKe79** [comment]
Hi,
this PR (d1421ea6698ae55e97bc5d172ca7c0e79141c2af) did not build correctly for all targets.
Following targets failed (1):
- OdroidXU3.arm

Logs and images for testing can be found here: http://nightly.builds.lakka.tv/PRs/pr_1175_000215/

_This is an automated message_

File: projects/OdroidXU3/options [hunk 1]
```diff
@@ -88,10 +88,10 @@
   # Xorg Graphic drivers to use (all / i915,i965,r200,r300,r600,nvidia,nouveau)
   # Space separated list is supported,
   # e.g. GRAPHIC_DRIVERS="i915 i965 r300 r600 radeonsi nvidia nouveau"
-    GRAPHIC_DRIVERS=""
+    GRAPHIC_DRIVERS="lima"
```
Comment (Ntemis) [inline projects/OdroidXU3/options:91]: Why you set this?

**Review by Ntemis — CHANGES_REQUESTED** [review]

File: projects/OdroidXU3/options (hunk 1 above)
Comment (by-gnome) [reply projects/OdroidXU3/options:91]: mesa failed with: Problem encountered: EGL requires dri.
So this needs some galium driver.

Comment (by-gnome) [reply projects/OdroidXU3/options:91]: https://github.com/libretro/Lakka-LibreELEC/blob/master/config/graphic#L67L70
https://github.com/mesa3d/mesa/blob/master/meson.build#L219L248

Comment (by-gnome) [reply projects/OdroidXU3/options:91]: Is this wrong?

Comment (Ntemis) [reply projects/OdroidXU3/options:91]: XU4 needs panfrost as is a midgard chip

**Comment by ToKe79** [comment]
Hi,
the build of this PR (5dec1fe5bfe706767eece772f0be03caf70f1444) finished successfully for all 20 targets.
Images for testing and logs are here: http://nightly.builds.lakka.tv/PRs/pr_1175_000216/

_This is an automated message_

File: projects/OdroidXU3/options (hunk 1 above)
Comment (by-gnome) [reply projects/OdroidXU3/options]: mesa removed.

**Pull request merged**